api/README.md

```diff
@@ -180,6 +180,8 @@ Returns the settings in JSON format.
 
 Request different types of data in various formats.
 
+Each of the export endpoints except contacts supports a `filters.human: true` parameter which returns date formatted in human readable form (ISO 8601). Setting this parameter to false or leaving it out will return dates formatted as an epoch timestamp.
```

Comment:
I don't think "filters" is the ideal parameter here. The API also takes "options" which makes more sense. Can we change it to `options.humanReadable: true` instead? You'll need to change the webapp Export service to take a third option, or modify `filters` to `params` so it can take, for example:


```js
Export('reports', { filters: {}, options: { humanReadable: true } });
```

Reply:
I went with having a third option parameter, looks much cleaner.